### PR TITLE
Resolve prisma error and Layout error (⚠️ potentially problematic)

### PR DESCRIPTION
**Pull request by abhishekrathi16**
Fixes issue #31 
This pull request will:

- [x] Fix errors related to tRPC and Prisma
- [x] Fix issues related to Layout

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (sutyum) [inline .vscode/launch.json]: do not commit your ide related files

File: .npmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+public-hoist-pattern[]=*prisma*
```
Comment (sutyum) [inline .npmrc:1]: why is this needed?

File: README.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 # VV+
 
 ## TODO
+
```
Comment (sutyum) [inline README.md]: unnecessary commit

File: apps/designer/postcss.config.cjs [hunk 4]
```diff
@@ -0,0 +1 @@
+module.exports = require('config/postcss.config');
```
Comment (sutyum) [inline apps/designer/postcss.config.cjs:1]: why `.cjs`?

File: apps/designer/src/env/client.mjs [hunk 5]
```diff
@@ -0,0 +1,33 @@
+// @ts-nocheck
```
Comment (sutyum) [inline apps/designer/src/env/client.mjs:1]: why are these files needed?

File: apps/designer/tailwind.config.cjs [hunk 6]
```diff
@@ -0,0 +1,3 @@
+import('tailwindcss').Config;
```
Comment (sutyum) [inline apps/designer/tailwind.config.cjs:1]: why is this `.cjs`?

File: tsconfig.json [hunk 7]
```diff
@@ -0,0 +1,14 @@
+{
+    "include": ["next.d.ts", "**/*.ts", "**/*.tsx"],
+    "baseUrl": ".",
+    "compilerOptions": {
+      //"declaration": false,
```
Comment (sutyum) [inline tsconfig.json:5]: why?

File: packages/db/tsconfig.json [hunk 8]
```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "../../tsconfig.json",
+    "compilerOptions": {
+        //"preserveSymlinks": true,
```
Comment (sutyum) [inline packages/db/tsconfig.json:4]: why?

**Review by sutyum — CHANGES_REQUESTED** [review]
- Why is `[apps/designer/.eslintrc.js → apps/designer/.eslintrc.cjs](https://github.com/TechnocultureResearch/VVplus/pull/32/files#diff-60d32cb98040c5e6543bbbe8855df6958c522895aeac39611b054048c17e0594)` needed?

**Comment by abhishekrathi16** [comment]
This pull request is linked to #30 which resolved layout issue and tRPC errors. The pull request created from newhouse branch faced some unexpected errors which forced me to create this pull request. The code here is exactly the same as that of newhouse branch since the branch was checked out from a verified commit. Kindly merge this pull request into the dev branch instead of #30 .

File: apps/designer/postcss.config.cjs (hunk 4 above)
Comment (abhishekrathi16) [reply apps/designer/postcss.config.cjs:1]: Faced a few errors like- 
Error [ERR_REQUIRE_ESM]: require() of ES Module /home/user/VVplus/apps/designer/postcss.config.js from /home/user/VVplus/node_modules/.pnpm/next@12.2.5_qtpcxnaaarbm4ws7ughq6oxfve/node_modules/next/dist/lib/find-config.js not supported.

postcss.config.js is treated as an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which declares all .js files in that package scope as ES modules.

Instead rename postcss.config.js to end in .cjs, change the requiring code to use dynamic import() which is available in all CommonJS modules, or change "type": "module" to "type": "commonjs" in /home/user/VVplus/apps/designer/package.json to treat all .js files as CommonJS (using .mjs for all ES modules instead).

So, we changed the file type to .cjs instead of .js

File: README.md (hunk 3 above)
Comment (abhishekrathi16) [reply README.md:4]: This file was throwing warning- "MD032/blanks-around-lists: Lists should be surrounded by blank lines". To resolve the warning, this change was made.

File: .vscode/launch.json (hunk 1 above)
Comment (abhishekrathi16) [reply .vscode/launch.json]: File has been deleted.

Comment (chandantiwari1) [reply .vscode/launch.json]: > * Why is `[apps/designer/.eslintrc.js → apps/designer/.eslintrc.cjs](https://github.com/TechnocultureResearch/VVplus/pull/32/files#diff-60d32cb98040c5e6543bbbe8855df6958c522895aeac39611b054048c17e0594)` needed?

https://github.com/nodejs/modules/issues/500
https://github.com/t3-oss/create-t3-turbo/blob/main/apps/nextjs/src/env/client.mjs
used in this repo ,suggested by you sir to follow this

File: .npmrc (hunk 2 above)
Comment (chandantiwari1) [reply .npmrc:1]: this generates error
https://pnpm.io/npmrc

File: README.md (hunk 3 above)
Comment (chandantiwari1) [reply README.md]: ok sir

File: apps/designer/postcss.config.cjs (hunk 4 above)
Comment (chandantiwari1) [reply apps/designer/postcss.config.cjs:1]: to import modules dynamically

File: packages/db/tsconfig.json (hunk 8 above)
Comment (chandantiwari1) [reply packages/db/tsconfig.json:4]: three tsconfig.json files are in our vvplus and we need to extends them to each other.
reference:https://github.com/t3-oss/create-t3-turbo/blob/main/apps/nextjs/tsconfig.json

File: tsconfig.json (hunk 7 above)
Comment (chandantiwari1) [reply tsconfig.json:5]: to fix the composite file error, need to declare a base url.

File: apps/designer/tailwind.config.cjs (hunk 6 above)
Comment (chandantiwari1) [reply apps/designer/tailwind.config.cjs:1]: referencee: https://github.com/t3-oss/create-t3-turbo/blob/main/apps/nextjs/tsconfig.json

File: apps/designer/src/env/client.mjs (hunk 5 above)
Comment (chandantiwari1) [reply apps/designer/src/env/client.mjs:1]: reference :https://github.com/t3-oss/create-t3-turbo/

other prisma and trpc won't work with the next.js and dynamic module import will not work

**Review by sutyum — APPROVED** [review]
